### PR TITLE
General Cleanup

### PR DESCRIPTION
**Pull request by alterationx10**
Condensed some pages into 1:
-  Service Modules Patterns
- ZIO JSON usage

Misc cleanup/formatting/spelling.

### CODE REVIEW

**Comment by alterationx10** [comment]
@tdrozdowski What do you think of these changes, and condensing some of the pages you wrote?

File: mdocs/zio-prelude/validation.md [hunk 1]
```diff
@@ -42,7 +38,7 @@ object MyValidations {
 }
 
 val ash = Student(23, "Ash")
-val shelia = Student(19, "Shelia")
+val Sheila = Student(19, "Sheila")
```
Comment (tdrozdowski) [inline mdocs/zio-prelude/validation.md:41]: Why cap Shelia val?

**Review by tdrozdowski — COMMENTED** [review]
Seems fine to combine things - only issue is there's no easy navigation to jump to topics at the end of a long page.  So I tried to break things up along those lines.  Not sure if we have a navigation solution for that...

File: mdocs/zio-prelude/validation.md (hunk 1 above)
Comment (alterationx10) [reply mdocs/zio-prelude/validation.md:41]: because Search and Replace all 😅 . Will fix.

**Comment by alterationx10** [comment]
@tdrozdowski 
> there's no easy navigation to jump to topics at the end of a long page.
Using the markdown headers ( i.e. ## Some Label) it generates a TOC style menu on the right

**Comment by alterationx10** [comment]
> Seems fine to combine things 

SOLD!

**Pull request merged**